script/Arcueid/Day 6/06_00_ARC06_2_1.txt

```diff
@@ -450,13 +451,13 @@ The past two days were really something. I'm sure I picked up some bumps and gra
 [sha:2616e47521fef00300421bcdacb505d66a9bb244]{
 -- Page 211, Offset 14420.
 -- 消毒薬はしみたが、貧血の目眩に比べれば大した事はなかった。消毒薬を塗るたびに、
-// Avoiding repetition
 The antiseptic stings, but it's nothing compared to the dizziness of my anemia. Plus, every time she daubs on some more―――
 }
 [sha:2d958309496daa81cd0d77c6ad05573c9a05a4c6]{
 -- Page 211, Offset 14421.
 -- 「わあ、男の子なんですねー」
-"Wow, you really are a man."
+// to be rechecked
+"Wow, so manly."
```

Comment:
I'd say make this into "Wow, so manly!" because right now the text makes it look very sarcastic. Otherwise, I support this tl